src/python/tests/conftest.py

```diff
@@ -64,3 +66,19 @@ def _json_data(file_name: str) -> Any:
         return get_json(data_dir / file_name)
 
     return _json_data
+
+
+@pytest.fixture(name="assert_files")
+def assert_files() -> Callable[[Path, Path], None]:
```

Comment:
```suggestion
def assert_files() -> Callable:
```
Just a suggestion, but I believe it may make things less complicated (and long).